cmd/run/run.go

```diff
@@ -394,32 +398,36 @@ func (s *ServerContext) datastoreConfig(config *serverconfig.Config) (storage.Op
 	var err error
 	switch config.Datastore.Engine {
 	case "memory":
+		// override for "memory" datastore
+		tokenSerializer = encoder.NewStringContinuationTokenSerializer()
```

Comment:
Is there any reason we can't use the same `NewSQLContinuationTokenSerializer` here?

Reply:
We can! I have it tested [here](https://github.com/openfga/openfga/pull/2020/files#diff-4b51ba45629b21e0486543c70a4671f3174aa81efc31290c89e174681814fe26R23-R33)